superset/databases/api.py

```diff
@@ -607,6 +608,7 @@ def test_connection(  # pylint: disable=too-many-return-statements
             TestConnectionDatabaseCommand(g.user, item).run()
             return self.response(200, message="OK")
         except DatabaseTestConnectionFailedError as ex:
+            logger.warning(ex)
```

Comment:
this log will read out the following:
`WARNING:superset.databases.api:Could not load database driver: mssql+pymssql`

Reply:
That is one possible error message - others are not so innocuous and include username and password information. We've seen that with a few different database engines.